scenarios/19c_The_Oracle.cfg

```diff
@@ -1674,7 +1678,12 @@
             side=1
             [modifications]
                 {TRAIT_LOYAL}
+#ifdef NIGHTMARE
```

Comment:
Quick is fine, but no weak please.

Reply:
ok, removed weak in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).